train_classification.py

```diff
@@ -0,0 +1,435 @@
+import time
```

Comment:
Why do you had back the training classification?

Reply:
oups thanks for pointing out. Got mixed up in my rebasing.